doc/topics/cloud/qs.rst

```diff
@@ -0,0 +1,128 @@
+.. _salt-cloud-qs:
+
+=====================
+Salt Cloud Quickstart
+=====================
+
+Salt Cloud is built-in to Salt and is configured on and executed from your Salt
+Master. On some platforms, Salt Cloud is installed by the ``salt-cloud``
```

Comment:
salt-cloud doesn't necessarily have to be run from the salt-master. It can be run anywhere it is installed and cloud configuration files are in place. While running salt-cloud commands from the same machine that the Salt Master is installed on is fairly typical, salt-cloud has it's own process and doesn't rely on the salt-master process. For example, you can still run salt-cloud commands from a masterless minion.